lib/CheckboxItem/CheckboxItemAccessibilitySupport.js

```diff
@@ -0,0 +1,30 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name CheckboxItemAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: ['checked']}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (props) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			this.setAttribute('role', enabled ? 'checkbox' : null);
+			this.setAttribute('tabindex', enabled ? 0 : null);
+			this.setAttribute('aria-checked', enabled ? String(this.checked) : null);
+			this.setAttribute('aria-labelledby', enabled ? this.getId() : null);
```

Comment:
We have more checked again about above aria-labelledby. 
ChromeVox does not read chekboxItem content when aria-labelledby is added to control's Id.
ChromeVox reads it well when aria-labelledby is only added to client's id. 
However, WebOS TV and Nexus5's screen reader read well in both case, so we will change aria-labelledby's value to client's id for consistency.